internal/template/funcmap.go

```diff
@@ -145,3 +146,13 @@ func regexReplaceAll(regex string, s string, repl string) (string, error) {
 func makeSlice(args ...interface{}) []interface{} {
 	return args
 }
+
+// schemaDescription returns a heredoc or single line string
+// according to the description provided format
+func schemaDescription(s string) string {
+	if strings.Contains(s, "\n") {
+		return fmt.Sprintf("<<EOF\n%s\nEOF", s)
```

Comment:
Maybe add one more \n at the end since the final EOF needs to be alone on its own line.

Reply:
Makes sense but that final `\n` is added on templating. I'll do my best to explain:

This is an example of how it's used:

```
1- ...
2- description = {{schemaDescription __schema__.properties.someProperty.description}}
3- ...
```

What you're suggesting is the natural linebreak that we intentionally put between line 2 and 3 in the template.

Here's an example of the output so you can validate everything is alright:

https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/964/files#diff-1d81e567868c331d72f2628f1287d393e95b80ccba397396aed80c84eb1292e2R19-R22

Reply:
Oh right. Templating whitespace is a bit weird since you can do `{{-` and `-}}` to eat whitespace, but by default it adds a newline because (as you said) we naturally add one as part of the template. Thanks for the details, resolving.